frontend/server/src/DAO/ProblemsetProblems.php

```diff
@@ -31,7 +31,8 @@ class ProblemsetProblems extends \OmegaUp\DAO\Base\ProblemsetProblems {
                 a.max_points,
                 p.alias AS problem_alias,
                 a.publish_time_delay,
-                p.problem_id
+                p.problem_id,
+                p.order as problem_order
```

Comment:
este order no es el orden del problema en el problemset:

https://github.com/omegaup/omegaup/blob/15e198d5e042bb69685164001a449d5b4b53ba73/frontend/database/schema.sql#L542

```suggestion
                pp.order as problem_order
```

según yo con esto se debería poder quitar el `intval(...)`  en frontend/server/src/Controllers/Course.php:711.